content/docs/tasks/security/vault-ca/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Istio Vault CA Integration
-description: Tutorial on how to plug in a Vault CA for issuing certificates in Istio.
+description: Tutorial on Istio Vault CA integration and a demo of Istio `mTLS` using certificates from a Vault CA.
```

Comment:
please replace 'mtls' with mutual TLS to be consistent with other tasks. Here and everywhere in this file

Reply:
Done.